src/components/List_Products.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import { saveItem } from '../lib/api';
+import { db } from '../lib/firebase';
+import { collection, onSnapshot } from 'firebase/firestore';
+
+function ListProducts() {
+  const [items, setItems] = useState(null);
```

Comment:
just for consistency. I see that for `itemName` the initial value is an empty string but for the array `items` the initial value is null. @FauricioValencia @alxmcr what do you guys think about this?

Reply:
[My thoughts](https://github.com/the-collab-lab/tcl-39-smart-shopping-list/pull/16/files#r785371614) ^^^!  😄